File: ZezeJava/Zege/client/src/Zeze/Builtin/LinkdBase/ModuleLinkdBase.java
package Zeze.Builtin.LinkdBase;

import Zeze.Transaction.Procedure;

public class ModuleLinkdBase extends AbstractModule {
    public void Start(Zege.App app) throws Throwable {
    }

    public void Stop(Zege.App app) throws Throwable {
    }

    @Override
    protected long ProcessReportError(Zeze.Builtin.LinkdBase.ReportError p) {
        System.out.println("ReportError code=" + p.Argument.getCode() + " desc=" + p.Argument.getDesc());
        p.getSender().close();
        return Procedure.Success;
    }

    // ZEZE_FILE_CHUNK {{{ GEN MODULE @formatter:off
    public ModuleLinkdBase(Zege.App app) {
        super(app);
    }
    // ZEZE_FILE_CHUNK }}} GEN MODULE @formatter:on
}

File: Zege/Zeze/Builtin/LinkdBase/ModuleLinkdBase.cs

using Zeze.Util;

namespace Zeze.Builtin.LinkdBase
{
    public partial class ModuleLinkdBase : AbstractModule
    {
        public void Start(global::Zege.App app)
        {
        }

        public void Stop(global::Zege.App app)
        {
        }

        protected override System.Threading.Tasks.Task<long> ProcessReportError(Zeze.Net.Protocol _p)
        {
            var p = _p as ReportError;
            return Task.FromResult(ResultCode.NotImplement);
        }

    }
}
